### ISSUE TITLE
Settlers Graphics

### ISSUE THREAD

**Issue by arpee** [SaitoTech/saito-lite-rust#2625]
#### General
 - [ ] Splash Background for invitations etc.
 - [ ] Invite Background

#### Bandit related
- [ ] New Bandit Icon
- [ ] New Bandit Card
- [ ] New Robin Hood Icon
- [ ] New Robin Hood Card
- [ ] New Soldier card to match
- [ ] New Soldier icon to match

#### Icons
- [ ] Trade Icon
- [ ] Bank Icon
- [ ] Dice Icon 
- [ ] Build Icon
- [ ] Pause Icon
- [ ] Go Icon
- [ ] Score Icon

#### Misc
- [ ] 'Slice' graphics for popup for each dev card.
    Now that the cards only just pop up behind the hud we need wide short versions of the dev crards, bandit and robin to feature there.

**Comment by LawHe2o** [comment]
**Dice Icon:**

![Image](https://github.com/user-attachments/assets/8b5ce2b9-56e2-4a40-9613-c55e31081865)
![Image](https://github.com/user-attachments/assets/42d58289-ab4f-4d80-99e3-b4696b4fbfc2)
![Image](https://github.com/user-attachments/assets/0f687f29-82f2-4210-bb26-0957234a978e)
![Image](https://github.com/user-attachments/assets/28558f82-5697-4c4c-895e-6b818f709b10)
![Image](https://github.com/user-attachments/assets/73888232-0009-4fa0-928b-4cbf4588e625)
![Image](https://github.com/user-attachments/assets/bae65695-e4c8-43f1-8c0b-74a541d5fab9)

**Action Icon:**

![Image](https://github.com/user-attachments/assets/231b0b21-66f6-4cc2-9312-46aff9e8e9b0)

**Bank Icon:**

![Image](https://github.com/user-attachments/assets/9377c44b-4b1c-4355-b4cd-f3ab21a0504d)

**Buid Icon:**

![Image](https://github.com/user-attachments/assets/38dc4787-ce4c-460b-bf7d-ebd1b255a50f)

**Score Icon:**

![Image](https://github.com/user-attachments/assets/3f4c84a2-0231-4880-b9a0-5600cb82004c)

**Play & Pause Icons:**

![Image](https://github.com/user-attachments/assets/d939d858-488b-47aa-8a6e-629f1dd51f50)
![Image](https://github.com/user-attachments/assets/3fe011f5-cb05-435f-88c6-21c30641000d)

**Settlers principal background:**
![Image](https://github.com/user-attachments/assets/a073d076-b21c-40b4-821e-1dc0e5f75c79)

Currently working on the robin, the bandit and the knight, trying new interesting compositions, with visual history and interest

**Comment by arpee** [comment]
For the dice icon - we need a roll dice icon - not representations of the dice... (have you played the game?)

Also these are not really doing much to improve on what we had. We want icons that match the theme, look like things from that era.
(play and pause possibly an exception).

**Comment by LawHe2o** [comment]
> For the dice icon - we need a roll dice icon - not representations of the dice... (have you played the game?)
> 
> Also these are not really doing much to improve on what we had. We want icons that match the theme, look like things from that era.
> (play and pause possibly an exception).
> 
> 

Yes sir I have played the game, just was thinking that is actually good to display the number with the dice and not just text, but I will make a "throw dice" icon instead. 

I will explore some options to make it look more older, medieval icons instead, maybe adding some cuts like we have in the card so they I'll look less polished.

**Comment by arpee** [comment]
We are wanting to replace the roll dice icon we have not to match the other new icons.... I am worried you are not aware of were these icons will be used...

**Comment by LawHe2o** [comment]
> We are wanting to replace the roll dice icon we have not to match the other new icons.... I am worried you are not aware of were these icons will be used...
> 

![IMG_20241021_235003_045.jpg](https://github.com/user-attachments/assets/fbeb64dc-715a-495d-ad34-40044882d883)

The new icons will be used there, right? 
Except for the roll the dice one, that is displayed in our turn, but then new icons are fine? Should only work on the roll dice?

**Comment by LawHe2o** [comment]
**New dice roll icon:**
![Image](https://github.com/user-attachments/assets/99222e51-2180-4420-9e15-04664cf2a8bc)

**Comment by LawHe2o** [comment]
**Robin new set:**

![Image](https://github.com/user-attachments/assets/ca40cea1-17be-4add-8615-cbd22e6e128e)
![Image](https://github.com/user-attachments/assets/7f150757-9902-4198-9f27-8d4745576987)
![Image](https://github.com/user-attachments/assets/4048575d-6012-40c9-96ae-996d190c821f)
![Image](https://github.com/user-attachments/assets/9a61d22f-3558-42bb-9f92-9f842d0ff595)
![Image](https://github.com/user-attachments/assets/081cd550-ffa0-4006-9f31-022b01050b60)
![Image](https://github.com/user-attachments/assets/e57eacc5-eca4-464d-994e-918b2c4dd835)
![Image](https://github.com/user-attachments/assets/eaa4856d-f4f9-4607-873d-e7820276db01)
![Image](https://github.com/user-attachments/assets/fd7c012e-9297-4150-991f-63a687cfc913)
![Image](https://github.com/user-attachments/assets/f922317d-69f0-463e-ae26-29d42c02587c)
![Image](https://github.com/user-attachments/assets/fac9bcb9-e620-47a2-9b01-1e33de310d37)
![Image](https://github.com/user-attachments/assets/659cc710-ad70-4c5f-90e6-be470894a5f6)
![Image](https://github.com/user-attachments/assets/74f30162-14ae-4dc5-9234-73505fe205b5)
![Image](https://github.com/user-attachments/assets/6463011b-d522-4d75-940c-1683bb370ce8)
![Image](https://github.com/user-attachments/assets/f7e9ffec-6595-418f-ac17-184cbb086f56)
![Image](https://github.com/user-attachments/assets/c4c31b6b-47e4-4dfd-97a8-626cccd551d3)
![Image](https://github.com/user-attachments/assets/afd420da-c1e9-4c85-beed-584864df769d)
![Image](https://github.com/user-attachments/assets/22289c80-f04e-4b45-b60a-0c3019449d89)
![Image](https://github.com/user-attachments/assets/a9f948f1-c102-45c8-8410-a720d3549b89)
![Image](https://github.com/user-attachments/assets/c8075903-268b-4045-8d16-19382e2badb7)
![Image](https://github.com/user-attachments/assets/b0cc0f0c-13ae-43fd-a767-3067aeffd9f8)
![Image](https://github.com/user-attachments/assets/f5395be6-1372-4e09-9a71-8255c092ac60)
![Image](https://github.com/user-attachments/assets/f1b4f848-877c-455b-83b6-a3d9b92b104e)
![Image](https://github.com/user-attachments/assets/fc899482-8a08-4df4-8f77-9005431e2957)
![Image](https://github.com/user-attachments/assets/273a0ac0-2148-45b0-b780-e2a772e2506f)
![Image](https://github.com/user-attachments/assets/35511156-5cb3-42c9-a0b4-84a950cfd606)
![Image](https://github.com/user-attachments/assets/466afeaa-3871-4268-90f3-742c68c9cf45)
![Image](https://github.com/user-attachments/assets/7ffca8e2-d65f-4efd-9fd5-98ee954a6446)
![Image](https://github.com/user-attachments/assets/f1408350-5ea1-43de-83e5-146201ea4c52)

**Comment by LawHe2o** [comment]
**Bandit new set:**

![Image](https://github.com/user-attachments/assets/0369da49-fc32-4bd3-b33c-6493f97e6467)
![Image](https://github.com/user-attachments/assets/302c9371-0d4c-467e-ba11-069cb60a48be)
![Image](https://github.com/user-attachments/assets/7e7dff8a-ca92-4407-859b-173000d0a909)
![Image](https://github.com/user-attachments/assets/90c1a33f-9554-4082-a7cb-a888f6252f62)
![Image](https://github.com/user-attachments/assets/0e202e62-9087-4da9-a771-fde5523b94e7)
![Image](https://github.com/user-attachments/assets/f0495a22-f95b-479f-b305-a3b60ebffb8e)
![Image](https://github.com/user-attachments/assets/15051331-748c-4f76-81b2-fd0a8ba61190)
![Image](https://github.com/user-attachments/assets/fd9ccfa7-9adb-4023-adec-46a71a876031)
![Image](https://github.com/user-attachments/assets/eaba93c3-aae5-4a39-b327-a2319315f2a1)
![Image](https://github.com/user-attachments/assets/fe02a472-2fd2-4193-84c7-52f3c4db1329)
![Image](https://github.com/user-attachments/assets/e8fe6387-3a03-4832-9327-f73893edd697)
![Image](https://github.com/user-attachments/assets/0557c768-2211-4797-9821-86b6ca563fab)
![Image](https://github.com/user-attachments/assets/b7449167-0ce4-43bb-a72a-ea67afd09713)
![Image](https://github.com/user-attachments/assets/13b499b0-4001-46f7-a2d1-89f90f83e42f)
![Image](https://github.com/user-attachments/assets/08b2dc7d-2ab6-4ced-9090-51a215b4f99f)
![Image](https://github.com/user-attachments/assets/d9a2392f-80af-4138-8b71-5af411f024ad)
![Image](https://github.com/user-attachments/assets/59ea07f4-1f13-4acc-8e4b-961b9b758c11)
![Image](https://github.com/user-attachments/assets/addbb799-74e9-4a6f-8eb0-a28add617ab9)
![Image](https://github.com/user-attachments/assets/15c16942-8f9c-4c3e-891e-3092a4b65cdb)
![Image](https://github.com/user-attachments/assets/b2f089c3-5720-4abf-99ce-0e6d09a8169c)
![Image](https://github.com/user-attachments/assets/1e341874-c071-42ec-b80b-1787c685e02c)
![Image](https://github.com/user-attachments/assets/f93906e8-f620-421e-bcc2-f6232e6caa90)
![Image](https://github.com/user-attachments/assets/599f259f-76b5-404b-bbcf-4bb2b3d5aa70)
![Image](https://github.com/user-attachments/assets/25ae4ea4-06af-4b2c-903b-56e03d58fbcb)
![Image](https://github.com/user-attachments/assets/043c6e11-84ad-44dd-b03c-3aca4781a2b0)
![Image](https://github.com/user-attachments/assets/b69217ce-0cd5-45f1-8110-0df1b00cc132)
![Image](https://github.com/user-attachments/assets/93ed42ec-6b75-4c70-b35d-54fce335d953)
![Image](https://github.com/user-attachments/assets/85e3788d-563b-455f-85ba-85e09a66b1fd)

**Comment by LawHe2o** [comment]
Soldier / Knight artwork:

[Soldier artworks here](https://imgur.com/a/SmtRjiL)

Arcade invite artwork:

[Invite](https://imgur.com/a/invite-artwork-VXFRlhj)

Upload them trough imgur cause it drops me an error when I try to upload them here.

**Comment by arpee** [comment]
So, some feedback - we don't need to see _so many_ versions of each graphic... none of these are particularly good. 

We need you to work hard on _an idea_ and present that idea with arguments about why it works. Then we can talk about it and make improvements. 

First thoughts - these pictures all scream AI, which is mostly because they are clearly short prompts and the AI is doing most of the work. You need a story about what is in the picture. Also for the game the bandit should hopefully look like someone who might steal a sheep or some lumber, maybe he has a wagon? Robin hood, maybe there is a merry man or two in the vicinity helping?

We want the cards to have character and to make a story that the game tells.

**Comment by arpee** [comment]
We also still don't have a 'roll the dice' icon.

**Comment by LawHe2o** [comment]
> **New dice roll icon:** ![Image](https://github.com/user-attachments/assets/99222e51-2180-4420-9e15-04664cf2a8bc)

@arpee 

Of course, I understand you, in reality the prompts that I have been using are quite long, I can attach them to you in the following spoilers, in any case I can continue trying to offer a better narrative for the following propmts, generating 5 or 6 images for each one , and present them to you again in the shortest amount of time possible, BUT making sure that they are of better quality, the soldiers, gentlemen, don't you think they are good either? And the invitation background? The dice icon is here Richard, does it work or do you have something else in mind, if so send me a small example, it would help me a lot.

> So, some feedback - we don't need to see _so many_ versions of each graphic... none of these are particularly good.
> 
> We need you to work hard on _an idea_ and present that idea with arguments about why it works. Then we can talk about it and make improvements.
> 
> First thoughts - these pictures all scream AI, which is mostly because they are clearly short prompts and the AI is doing most of the work. You need a story about what is in the picture. Also for the game the bandit should hopefully look like someone who might steal a sheep or some lumber, maybe he has a wagon? Robin hood, maybe there is a merry man or two in the vicinity helping?
> 
> We want the cards to have character and to make a story that the game tells.

Of course, I understand you, in reality the prompts that I have been using are quite long, in any case I can continue trying to offer a better narrative for the following propmts, generating 3 or 4 images for each one , and present them to you again in the shortest amount of time possible, BUT making sure that they are of better quality, the soldiers, knights, don't you think they are good either? And the invitation background? The dice icon is here Richard, does it work or do you have something else in mind, if you can send me a small example, it would help me a lot, also the dice icon is there, I attach it here as well:

![Image](https://github.com/user-attachments/assets/95321f99-fc23-409d-a4ba-3436d56d7d1b)

**Comment by LawHe2o** [comment]
Hey guys, Update here:

Icon set:
![icons set](https://github.com/user-attachments/assets/23c87107-dbc0-41da-a8da-519f3594166d)
New possible layout for the cards:
![cards layouts](https://github.com/user-attachments/assets/591b2fac-f49a-473b-a559-34081ebef0aa)}
Posters:
![Satoia poster](https://github.com/user-attachments/assets/cde8684d-b395-4403-a401-c7227a4f65aa)
![satoia vertical](https://github.com/user-attachments/assets/a8039046-0970-4a54-bd4f-a91d1a0ca99d)
Card art:
![baazar](https://github.com/user-attachments/assets/8764c444-a45d-4fcb-8d7a-c5e9b6c76258)
![baazar_2](https://github.com/user-attachments/assets/de5daf63-c9a8-4905-8a8a-44c7e8c57b85)
![brew_house](https://github.com/user-attachments/assets/188e7dc9-7cbc-49b6-8ed3-1a28065c2ccf)
![brew_house_2](https://github.com/user-attachments/assets/4fb2901f-9c56-44fe-bb4a-81b02fb798a5)
![brew_house_3](https://github.com/user-attachments/assets/e3b6a72e-9d6f-4c46-96ce-6ec31b8eeeda)
![cathedral](https://github.com/user-attachments/assets/aeeb4470-e9a4-47b8-a6e3-1c48490054a4)
![cathedral_2](https://github.com/user-attachments/assets/312068f0-8c71-4941-bed1-4a85a7838d1c)
![chimestry](https://github.com/user-attachments/assets/4f8a73c8-0cb0-47fa-aa10-5fc1f8a6fa3f)
![chimestry_2](https://github.com/user-attachments/assets/a6efb785-67cc-4ab2-9c4b-5992b94575c6)
![chimestry_3](https://github.com/user-attachments/assets/46ae746e-218d-4b7f-95bd-919edd8c8dce)
![church](https://github.com/user-attachments/assets/8d1d774c-ab4e-40d1-a20a-5d501ef565a8)
![factory](https://github.com/user-attachments/assets/bd5253e0-f537-467f-8cef-ac6614498a84)
![factory_2](https://github.com/user-attachments/assets/1d2d1b5a-2ee3-4a74-89ff-8761cceaf429)
![lighthouse](https://github.com/user-attachments/assets/9cf2da00-aa6a-413c-a52f-50d8673580bc)
![lighthouse_2](https://github.com/user-attachments/assets/60ab0acc-e87d-4d4a-a00f-4e731aa186c9)
![lighthouse_3](https://github.com/user-attachments/assets/f8f9340c-d70c-4391-a1ab-43fdfe54663c)
![monopoly](https://github.com/user-attachments/assets/ee4055cb-e198-44f9-94e3-5954c63362b6)
![monopoly_2](https://github.com/user-attachments/assets/3040c957-a3de-4a10-81a0-d7572a23c64f)
![monopoly_3](https://github.com/user-attachments/assets/a9b8b87d-2048-4873-af3c-f0331116c1e0)
![port](https://github.com/user-attachments/assets/5222f919-a100-41e9-a9ec-485df1ec279b)
![port_2](https://github.com/user-attachments/assets/c17f110b-11e8-47d0-9e78-c77dc33aab0b)
![port_3](https://github.com/user-attachments/assets/d1637c78-c49a-4c46-9b3f-d5fae9214e9c)
![road_building](https://github.com/user-attachments/assets/4eba8f84-3fec-4b73-9650-ab460018f7be)
![road_building_2](https://github.com/user-attachments/assets/35f9609b-1824-44ef-8e84-3d57981fe1af)
![road_building_3](https://github.com/user-attachments/assets/1c690a08-9b47-49ad-94e2-f9c658cea3ad)
![roadbuilding_2](https://github.com/user-attachments/assets/c5147272-3f50-47fa-a1b8-f4e386dbfe5e)
![robin_hood](https://github.com/user-attachments/assets/b4993fcd-4809-46cc-ac6a-c1a52c46fbbb)
Still working in some of them like university, robin and windfall, not sure if those are all the cards, @arpee when you are free you're feedback is well recieved, also if you can provide me the link with all the cards, so I'm sure I do not miss anyone.
